command/texttools.js

```diff
@@ -23,17 +23,24 @@ const texttools = {
   cleanText: function(htmlStr) {
     // Remove the HTML marks.
     let strippedHtml = htmlStr.replace(/<[^>]+>/g, ' ');
+
     // Now replace the quotes and other markups.
     strippedHtml = strippedHtml
-      .replace(/&amp;/g, '&')
+      .replace(/&amp;/g, ' and ')
       .replace(/&rdquo;/g, '"')
       .replace(/&ldquo;/g, '"')
       .replace(/&rsquo;/g, "'")
       .replace(/&lsquo;/g, "'")
       .replace(/&mdash;/g, '-')
       .replace(/&ndash;/g, '-')
       .replace(/&nbsp;/g, ' ')
-      .replace(/&thinsp;/g, '');
+      .replace(/&thinsp;/g, '')
+      .replace(/&Aring;/g, 'A');
```

Comment:
Is there something about `&Aring;` that won't be handled by the catch-all on line 42 below? If not, I'd suggest leaving it out here since there are so many other characters that aren't getting the special treatment.

Reply:
Yeah, good point.